.golangci.yml

```diff
@@ -151,84 +213,171 @@ linters-settings:
   tenv:
     all: true
 
-  #unparam:
-  #  check-exported: true
+  testifylint:
+    enable-all: true
+    # TODO: Do not disable any options
+    disable:
+      - go-require
+
+  testpackage:
+    skip-regexp: "^$"
+
+  unparam:
+    # NOTE: Set this option to false if other projects rely on this project's code
+    check-exported: false
+
+  unused:
+    # TODO: Uncomment these two lines
+    # parameters-are-used: false
+    # local-variables-are-used: false
+    # NOTE: Set these options to true if other projects rely on this project's code
+    field-writes-are-uses: true
+    # exported-is-used: true # TODO: Fix issues with this option (upstream)
+    exported-fields-are-used: true
+
+  usestdlibvars:
+    http-method: true
+    http-status-code: true
+    time-weekday: false # TODO: Set to true
+    time-month: false # TODO: Set to true
+    time-layout: false # TODO: Set to true
+    crypto-hash: true
+    default-rpc-path: true
+    os-dev-null: true
+    sql-isolation-level: true
+    tls-signature-scheme: true
+    constant-kind: true
+    syslog-priority: true
 
   wrapcheck:
     ignorePackageGlobs:
       - github.com/gofiber/fiber/*
       - github.com/valyala/fasthttp
 
 issues:
-   exclude-use-default: false
+  exclude-use-default: false
+  exclude-case-sensitive: true
+  max-issues-per-linter: 0
+  max-same-issues: 0
+  exclude-rules:
+    - linters:
+        - goerr113
+      text: 'do not define dynamic errors, use wrapped static errors instead*'
+  # fix: true
 
 linters:
-  disable:
-    - spancheck # opentelemetry, irrelevant
-    - tagalign  # requires awkward manual formatting of struct tags
   enable:
     - asasalint
     - asciicheck
     - bidichk
     - bodyclose
     - containedctx
     - contextcheck
+    # - cyclop
+    - deadcode
+    - decorder
     - depguard
     - dogsled
+    # - dupl
+    # - dupword # TODO: Enable
     - durationcheck
     - errcheck
     - errchkjson
     - errname
     - errorlint
     - execinquery
     - exhaustive
+    # - exhaustivestruct
```

Comment:
Exhaustivestruct is deprecated in favor of exhaustruct

Reply:
Same as all other deprecated linters: I'd leave them on just in case they catch things the other linters do not catch.